### DISCUSSION TITLE
rights + license

### DISCUSSION THREAD

**Discussion by briesenberg07** [uwlib-cams/sinopia_maps#125] [seeking feedback]
We've used the [Creative Commons public domain dedication](https://creativecommons.org/publicdomain/zero/1.0/) for much previous CAMS web publishing. Is it the right license for the map_storage + sinopia_maps repositories and GitHub Pages?

I wonder this for two reasons:
- RDA Registry is offered with a [Attribution 4.0 International (CC BY 4.0)](https://creativecommons.org/licenses/by/4.0/deed.en_US) license, and our work is built using data published there.
- We may wish to receive attribution for derivative works?

**Comment by gerontakos** [comment]
We can't decide this; it's the property of UW created by UW Libraries. If you want to make the case for cc-by, you should appeal to the head of UWL CaMS, who may push the decision up the ladder themselves. I believe there are three licenses that support open data (and, therefore, our only choices): cc-0, cc-by, cc-by-sa. Another belief: the following organizations offer up their data on the web as follows:
wikidata.cc-by-sa
schema.org.cc-by-sa
rda registry.cc-by
ifla lrm.cc-by
new york times.cc-by
uw libraries.cc-0
sinopia.cc-0
Yale (Lux).cc-0
Of course everything is cc-0 if the owners aren't willing to take it to the courts.

**Reply by briesenberg07** [reply]
Thanks for weighing in and for the information @gerontakos !  
I don't actually remember the decision-making process we used when we initially applied CC-zero to UWLSWD datasets. Did we have a discussion with Erin at that time?

**Reply by gerontakos** [reply]
We did. We made a recommendation and she accepted it. I don't know if I ever knew if she went up-the-ladder for approval. We wanted originally to go cc-by-nc but that's not what we recommended to EG. Anyway, it was informal. 

The Libraries makes its web pages cc-by-nc.

The Libraries used to have a Licensing Work Group (Sion, Faye, Morag, Frances, Gordon, Corey, and Denise). Decisions were made, but I seriously doubt "linked open data" was discussed: it's a special problem area. 

I don't want to make a project of this (you could!). I'm happy to slap a license, either cc-0, cc-by, cc-by-sa -- or even cc-by-nc. If you don't want to approach Erin on this, I can. I'd recommend cc-0 until we have something that seems to require otherwise. As for the RDA being cc-by, I believe we only need provide them attribution. The responsibility for determining a chain of attribution falls on those who use our data: they see the attribution, they also provide the attribution to RDA -- because we fulfilled our obligation to attribute RDA. If anyone whose data we repurpose needs us to share alike, they have to slap an "sa" on their license.

**Reply by briesenberg07** [reply]
Thanks again for this.  
Yes, my main concern is whether or not it's okay to offer our MAPs under CC-zero, if they are built from the data at the RDA Registry (along with structure and content that we create), when the Registry is CC-By.  
I think my project will be to add an attribution for the Registry to our HTML pages, while keeping the CC-zero license for those pages as-is.

**Reply by briesenberg07** [reply]
See #126

**Comment by briesenberg07** [comment]
Closing discussion following:
- Discussion with UW Scholarly Publishing Outreach Librarian regarding application of CC-zero to pages presenting content originating in the RDA Registry - see [2023_06_rights_for_rda_pages](https://uwnetid.sharepoint.com/:f:/s/og_library_cams/EtLvifTPNChGmn5MEjMPfnMBWcp30nlywisvD62HqOLH9w?e=Bah09R)\* 
- Application of updated CC-zero footer, including attribution for the RDA Registry, to all sinopia_maps HTML pages presenting RDA data -- see  https://github.com/uwlib-cams/sinopia_maps/issues/126
- Application of updated CC-zero footer, including attribution for the RDA Registry, to map_storage HTML pages presenting HTML data -- see https://github.com/uwlib-cams/map_storage/issues/42

\**UW NetID required for access*

**Reply by briesenberg07** [reply]
And!

- Similar application to webviews pages! See https://github.com/uwlib-cams/webviews/issues/9
